### DISCUSSION TITLE
Mkdocs material documentation

### DISCUSSION THREAD

**Discussion by gabrielbdornas** [infovote-io/dashboard#10] [Show and tell]
@danybonfil, the [Mkdocs material](https://squidfunk.github.io/mkdocs-material/getting-started/) is a python static site generator where we could build our first prototype, alongside with [mkdocs-charts-plugin](https://timvink.github.io/mkdocs-charts-plugin/).

**Comment by danybonfil** [comment]
Both Mkdocs material and charts plugin where installed in the virtual environment by including them in the requirements.txt file (lines 1 and 6):

<img width="335" alt="image" src="https://github.com/infovote-io/dashboard/assets/43021461/f3e593d4-7e6d-40b2-82bd-24e2d2aa3144">
